x-pack/legacy/plugins/siem/public/components/load_more_table/index.tsx

```diff
@@ -213,7 +221,13 @@ export class LoadMoreTable<T, U, V, W, X, Y, Z, AA, AB> extends React.PureCompon
           )}
 
           <HeaderPanel
-            subtitle={`${i18n.SHOWING}: ${headerCount.toLocaleString()} ${headerUnit}`}
+            id={id}
+            showInspect={this.state.showInspect}
+            subtitle={
+              <>
+                <div>{`${i18n.SHOWING}: ${headerCount.toLocaleString()} ${headerUnit}`}</div>
```

Comment:
Take it you can't go without using the `div` here? Would be nice if it only needed the fragment or better yet just a string being returned?

If not, ignore this.